.github/workflows/unit-tests-check.yml

```diff
@@ -48,6 +48,9 @@ jobs:
         pip install setuptools wheel
         pip install -r requirements/edx/testing.txt
 
+    - name: Install Django 2.2
+      run: pip install django==2.2
```

Comment:
I'd rather add `-r requirements/edx/django.txt` to the `pip install` command above instead; we have that file largely for cases like this.

Reply:
Done